Ascend.java

```diff
@@ -1,7 +1,28 @@
 // Generates three integer random numbers in a given range,

```

Comment:
OVERALL:
The logic is very good, there are several issues regarding styling. Please review the styling guidelines. 
The main issue in the assignment is the first statement in Ascend.java this is a violation of HW1 guidelines. 
Next time, it will cause deduction of points.